.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,100 @@
+name: Bug Report
+description: Report a bug on Minehut
+title: "[BUG]: "
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Minehut bug report guide
```

Comment:
What part of it do you want removed? The entire front section? Because that is what is at the front of every issue request in the current forms. Inside the arrows. No one can put anything in there, its just text. If you still want it removed, feel free to correct me.

Reply:
Yes - the entire front section can be removed since it is used to explain what github forms provide.

Reply:
There you go then